PWGJE/TableProducer/jettaggerhf.cxx

```diff
@@ -115,7 +115,7 @@ struct JetTaggerHFTask {
   }
   PROCESS_SWITCH(JetTaggerHFTask, processMCD, "Fill tagging decision for mcd jets", false);
 
-  void processTraining(JetCollision const& collision, JetTableMCD const& mcdjets, JetTagTracksMCD const& tracks)
+  void processTraining(JetCollision const& /*collision*/, JetTableMCD const& /*mcdjets*/, JetTagTracksMCD const& /*tracks*/)
```

Comment:
these can be removed fully instead of commented out

Reply:
I kept them because the function body is empty so I guess, you will need the parameters when you implement it.